README.md

```diff
@@ -1 +1,175 @@
\ No newline at end of file
+# cms-ars-5.0-k8s-cluster-stig-overlay
+
+InSpec profile to validate the secure configuration of Kubernetes cluster against [DISA's](https://public.cyber.mil/stigs/) Kubernetes Secure Technical Implementation Guide (STIG) Version 1 Release 1 tailored for CMS ARS 5.0.
+
+## Getting Started  
+### InSpec (CINC-auditor) setup
+For maximum flexibility/accessibility, we’re moving to “cinc-auditor”, the open-source packaged binary version of Chef InSpec, compiled by the CINC (CINC Is Not Chef) project in coordination with Chef using Chef’s always-open-source InSpec source code. For more information: https://cinc.sh/
+
+It is intended and recommended that CINC-auditor and this profile overlay be run from a __"runner"__ host (such as a DevOps orchestration server, an administrative management system, or a developer's workstation/laptop) against the target. This can be any Unix/Linux/MacOS or Windows runner host, with access to the Internet.
+
+__For the best security of the runner, always install on the runner the _latest version_ of CINC-auditor.__ 
+
+__The simplest way to install CINC-auditor is to use this command for a UNIX/Linux/MacOS runner platform:__
+```
+curl -L https://omnitruck.cinc.sh/install.sh | sudo bash -s -- -P cinc-auditor
+```
+
+__or this command for Windows runner platform (Powershell):__
+```
+. { iwr -useb https://omnitruck.cinc.sh/install.ps1 } | iex; install -project cinc-auditor
+```
+To confirm successful install of cinc-auditor:
+```
+cinc-auditor -v
+```
+> sample output:  _4.24.32_
+
+Latest versions and other installation options are available at https://cinc.sh/start/auditor/.
+
+### Requirements
+
+#### Kubernetes Cluster
+- Kubernetes Platform deployment
+- Access to the Kubernetes Cluster API
+- Kubernetes Cluster Admin credentials cached on the runner.
+
+#### Install InSpec Kubernetes Train
+Kubernetes Train allows InSpec to send request over Kubernetes API to inspect the Kubernetes Cluster.
+
+```sh
+# Use one of the two following approaches for installing train-kubernetes.
+
+# if InSpec was installed as a gem, use the system gem binary to install train-kubernetes.
+# to check, compare `which inspec` to $GEM_HOME, if they match use
+gem install train-kubernetes -v 0.1.6
+
+# if InSpec was installed as a package, use the embedded gem binary to install train-kubernetes.
+# to check, compare `which inspec` to $GEM_HOME, if they do not match or if $GEM_HOME is null use
+sudo /opt/cinc-auditor/embedded/bin/gem install train-kubernetes -v 0.1.6
+
+# Import gem as InSpec plugin
+cinc-auditor plugin install train-kubernetes
+
+#If it has the version set to "= 0.1.6", modify it to "0.1.6" and save the file.
+vi ~/.inspec/plugins.json
+
+# Run the following command to confirm train-kubernetes is installed
+cinc-auditor plugin list
+```
+### How to execute this instance  
+(See: https://www.inspec.io/docs/reference/cli/)
+
+#### Validate access to Kubernetes API
+```sh
+kubectl get nodes
```

Comment:
Would it be helpful to insert the `Kubernetes API` instructions from EKS README before the `Validate access to Kubernetes API` section?

> ### Kubernetes API
> 
> The profile makes use of the `kubectl` utility to access the Kubernetes API. The runner host must have `kubectl` installed -- see the [Kubernetes documentation for tools](https://kubernetes.io/docs/tasks/tools/) for details.
> 
> A host's connection to the Kubernetes API is established using credentials recorded in the `kubeconfig` file. For the profile to use the Kubernetes API, the runner host must either have a valid `kubeconfig` file either in the default location ($HOME/.kube/config) or have designated a file as the `kubeconfig` file using the `$KUBECONFIG` environment variable. See the [Kubernetes documentation for kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) for details.
> 
> You can test if the runner host has access to the Kubernetes API by running `kubectl` from the command line:
> 
> ```
> kubectl get nodes
> ```
> 
> If the command returns a list of information on the EC2 nodes that comprise the cluster, then `kubectl` is able to support the profile.

Reply:
yes. done.